redhat-cop/infra-ansible
# adding documentation for the role: config-vnc-server

### What does this PR do?
This PR adds documentation for the config-vnc-server role. It also gets rid of an instance where latest was used instead of installed when checking for packages.

### How should this be tested?
Just read the README.md and make sure that the change to get rid of latest doesn't cause any complications with the playbook

### Is there a relevant Issue open for this?
#227 - for converting from latest to installed

### Other Relevant info, PRs, etc.

### People to notify
cc: @redhat-cop/infra-ansible @oybed @logandonley @tylerauerbeck
